<ui version="4.0" >
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Synchronization interval:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QSpinBox" name="SynchronizationIntervalSpinBox" >
         <property name="suffix" >
          <string>s</string>
         </property>
         <property name="minimum" >
          <number>120</number>
         </property>
         <property name="maximum" >
          <number>604800</number>
         </property>
         <property name="singleStep" >
          <number>30</number>
         </property>
         <property name="value" >
          <number>180</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Number of tweet per page:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QSpinBox" name="TweetsPerPageSpinBox" >
         <property name="minimum" >
          <number>1</number>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="singleStep" >
          <number>5</number>
         </property>
         <property name="value" >
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>Number of users per page:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QSpinBox" name="UsersPerPageSpinBox" >
         <property name="minimum" >
          <number>1</number>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="singleStep" >
          <number>5</number>
         </property>
         <property name="value" >
          <number>15</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QCheckBox" name="ShowMainWindowOnStartupCheckBox" >
         <property name="text" >
          <string>Show main window at startup.</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Notifications</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" >
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6" >
           <item>
            <widget class="QCheckBox" name="NotifyOnNewTweetCheckBox" >
             <property name="text" >
              <string>Notification on new tweet in timeline</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="NotifyOnNewReplyCheckBox" >
             <property name="text" >
              <string>Notification on new reply</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="NotifyOnNewDirectMessageCheckBox" >
             <property name="text" >
              <string>Notification on new direct message</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7" >
           <item>
            <widget class="QCheckBox" name="NotifyOnNewFollowerCheckBox" >
             <property name="text" >
              <string>Notification on new follower</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="NotifyOnNewFriendCheckBox" >
             <property name="text" >
              <string>Notification on new friend</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="NotifyOnSyncCompletedCheckBox" >
             <property name="text" >
              <string>Notification on synchronization completed</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Blacklist users</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2" >
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Whitelisted users</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="WhitelistedUserListWidget" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3" >
            <item>
             <spacer name="verticalSpacer_2" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="BlackListUserToolButton" >
              <property name="text" >
               <string>...</string>
              </property>
              <property name="icon" >
               <iconset resource="ressources.qrc" >
                <normaloff>:/icons/icons/tango/22x22/actions/go-next.png</normaloff>:/icons/icons/tango/22x22/actions/go-next.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="WhiteListUserToolButton" >
              <property name="text" >
               <string>...</string>
              </property>
              <property name="icon" >
               <iconset resource="ressources.qrc" >
                <normaloff>:/icons/icons/tango/22x22/actions/go-previous.png</normaloff>:/icons/icons/tango/22x22/actions/go-previous.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout" >
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Blacklisted users</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="BlacklistUserListWidget" />
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" >
         <item>
          <widget class="QGroupBox" name="groupBox" >
           <property name="title" >
            <string>Blacklisted tags</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4" >
            <item>
             <widget class="QListWidget" name="BlacklistTagListWidget" />
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4" >
              <item>
               <widget class="QLineEdit" name="BlacklistTagLineEdit" />
              </item>
              <item>
               <widget class="QPushButton" name="AddBlackListTagButton" >
                <property name="text" >
                 <string>Add</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2" >
           <property name="title" >
            <string>Whitelisted tags</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5" >
            <item>
             <widget class="QListWidget" name="WhitelistTagListWidget" />
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" >
              <item>
               <widget class="QLineEdit" name="WhitelistTagLineEdit" />
              </item>
              <item>
               <widget class="QPushButton" name="AddWhiteListTagButton" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Add</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Theme</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10" >
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Theme information</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7" >
          <item>
           <widget class="QSplitter" name="splitter" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QScrollArea" name="scrollArea" >
             <property name="widgetResizable" >
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents" >
              <property name="geometry" >
               <rect>
                <x>0</x>
                <y>0</y>
                <width>456</width>
                <height>143</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2" >
               <item>
                <widget class="QLabel" name="PreviewLabel" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text" >
                  <string>preview</string>
                 </property>
                 <property name="alignment" >
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
            <widget class="QTextBrowser" name="ThemeDescriptionTextBrowser" >
             <property name="openLinks" >
              <bool>false</bool>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Select theme</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="ThemeListWidget" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressources.qrc" />
 </resources>
 <connections/>
</ui>
